proto/dalc/dalc.proto

```diff
@@ -25,7 +26,7 @@ message SubmitBlockResponse {
 }
 
 message CheckBlockAvailabilityRequest {
-	optimint.Header header = 1;
+	uint64 height = 1;
```

Comment:
Optimint needs to keep optinint->celestia height mapping. This needs to be Celestia height.

Reply:
Does anything need to change here? I believe it is using this height directly when sampling. Should we call it da_layer_height instead?